CMakeLists.txt

```
@@ -106,9 +106,10 @@ set( WCSIM_CHECK_GEOMETRY_OVERLAPS $<IF:$<BOOL:${WCSim_Geometry_Overlaps_CHECK}>
 
 pbuilder_install_files(${CMAKE_INSTALL_PREFIX}/data
         ${PROJECT_SOURCE_DIR}/data/MuonFlux-HyperK-ThetaPhi.dat
-	${PROJECT_SOURCE_DIR}/data/biasprofile.dat
-	${PROJECT_SOURCE_DIR}/data/LightInjectors.json
-	${PROJECT_SOURCE_DIR}/data/LI_OD_laser_ball_photon_positions.csv
+	      ${PROJECT_SOURCE_DIR}/data/biasprofile.dat
+        ${PROJECT_SOURCE_DIR}/data/DSNBFluxes.json
+      	${PROJECT_SOURCE_DIR}/data/LightInjectors.json
+      	${PROJECT_SOURCE_DIR}/data/LI_OD_laser_ball_photon_positions.csv
```

Make whitespace consistent
```suggestion
	${PROJECT_SOURCE_DIR}/data/biasprofile.dat
        ${PROJECT_SOURCE_DIR}/data/DSNBFluxes.json
      	${PROJECT_SOURCE_DIR}/data/LightInjectors.json
      	${PROJECT_SOURCE_DIR}/data/LI_OD_laser_ball_photon_positions.csv
```